modules/oncodriveclustl/1.0.1/config/default.yaml

```diff
@@ -0,0 +1,63 @@
+lcr-modules:
+    
+    oncodriveclustl:
+
+        dirs:
+            _parent:
+
+        inputs:
+            samples:
+            maf_path: # Leave blank if using sample subset
+
+        scratch_subdirectories: [] # Recommended: mutect2
+
+        options:
+            mode: pathology  # pathology, samples, subset, maf           
+            genome_build: "grch37"
+            seq_type: ["genome","capture"]
+            
+            pathology: [["DLBCL","BL"],["FL","MCL","CLL"]]
```

Comment:
This is going to require `pathology` to be a column in the input metadata frame. Could you please add a `pathology-1.0.yaml` schema to the `schema` directory in your module directory? There's an example in the `star/1.4` module for how to do this.